CompletedProcess(args=['python', 'temp_test_case.py'], returncode=1, stdout=b'', stderr=b'...FE..F..................E.......E.........F...E...............E......E......E........E.F..F.......\n======================================================================\nERROR: test12 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 84, in test12\n    self.assertEqual(patched_source(input_12), original_source(input_12))\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_bug_qb.py", line 4, in original_func\n    (d1, d2, d3) = map(int, args[0].split())\nValueError: too many values to unpack (expected 3)\n\n======================================================================\nERROR: test32 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 204, in test32\n    self.assertEqual(patched_source(input_32), original_source(input_32))\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_bug_qb.py", line 4, in original_func\n    (d1, d2, d3) = map(int, args[0].split())\nValueError: too many values to unpack (expected 3)\n\n======================================================================\nERROR: test4 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 36, in test4\n    self.assertEqual(patched_source(input_4), original_source(input_4))\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_bug_qb.py", line 4, in original_func\n    (d1, d2, d3) = map(int, args[0].split())\nValueError: too many values to unpack (expected 3)\n\n======================================================================\nERROR: test52 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 324, in test52\n    self.assertEqual(patched_source(input_52), original_source(input_52))\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_bug_qb.py", line 4, in original_func\n    (d1, d2, d3) = map(int, args[0].split())\nValueError: too many values to unpack (expected 3)\n\n======================================================================\nERROR: test67 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 414, in test67\n    self.assertEqual(patched_source(input_67), original_source(input_67))\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_bug_qb.py", line 4, in original_func\n    (d1, d2, d3) = map(int, args[0].split())\nValueError: too many values to unpack (expected 3)\n\n======================================================================\nERROR: test73 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 450, in test73\n    self.assertEqual(patched_source(input_73), original_source(input_73))\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_bug_qb.py", line 4, in original_func\n    (d1, d2, d3) = map(int, args[0].split())\nValueError: too many values to unpack (expected 3)\n\n======================================================================\nERROR: test8 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 60, in test8\n    self.assertEqual(patched_source(input_8), original_source(input_8))\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_bug_qb.py", line 4, in original_func\n    (d1, d2, d3) = map(int, args[0].split())\nValueError: too many values to unpack (expected 3)\n\n======================================================================\nERROR: test88 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 540, in test88\n    self.assertEqual(patched_source(input_88), original_source(input_88))\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_bug_qb.py", line 4, in original_func\n    (d1, d2, d3) = map(int, args[0].split())\nValueError: too many values to unpack (expected 3)\n\n======================================================================\nFAIL: test11 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 78, in test11\n    self.assertEqual(patched_source(input_11), original_source(input_11))\nAssertionError: Lists differ: [34] != [37]\n\nFirst differing element 0:\n34\n37\n\n- [34]\n?   ^\n\n+ [37]\n?   ^\n\n\n======================================================================\nFAIL: test15 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 102, in test15\n    self.assertEqual(patched_source(input_15), original_source(input_15))\nAssertionError: Lists differ: [36] != [39]\n\nFirst differing element 0:\n36\n39\n\n- [36]\n?   ^\n\n+ [39]\n?   ^\n\n\n======================================================================\nFAIL: test49 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 306, in test49\n    self.assertEqual(patched_source(input_49), original_source(input_49))\nAssertionError: Lists differ: [40] != [45]\n\nFirst differing element 0:\n40\n45\n\n- [40]\n?   ^\n\n+ [45]\n?   ^\n\n\n======================================================================\nFAIL: test9 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 66, in test9\n    self.assertEqual(patched_source(input_9), original_source(input_9))\nAssertionError: Lists differ: [80] != [90]\n\nFirst differing element 0:\n80\n90\n\n- [80]\n?  ^\n\n+ [90]\n?  ^\n\n\n======================================================================\nFAIL: test92 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 564, in test92\n    self.assertEqual(patched_source(input_92), original_source(input_92))\nAssertionError: Lists differ: [104] != [152]\n\nFirst differing element 0:\n104\n152\n\n- [104]\n+ [152]\n\n----------------------------------------------------------------------\nRan 100 tests in 0.007s\n\nFAILED (failures=5, errors=8)\n')